regression/cbmc/Quantifiers1/quantifier-with-function-call.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+quantifier-with-function-call.c
+
+^EXIT=134$
```

Comment:
You'll have to add this one to `regression/validate-trace-xml-schema/check.py` as the failing invariant check implies the overall output isn't valid XML. But if you pick up my suggestion below then this will turn into `EXIT=6` and valid XML will be produced.